systemtests/src/test/java/io/enmasse/systemtest/common/certs/CertProviderTest.java

```diff
@@ -0,0 +1,426 @@
+
+        testConsole(webClient);
+    }
+
+    @Test
+    @Disabled("Disabled due to #2427")
+    void testConsoleCertBundleTrustCaCert() throws Exception {
+        CertBundle certBundle = createCertBundle();
+
+        createTestEnv(new CertSpecBuilder()
+                .withProvider(CertProvider.certBundle.name())
+                .withTlsKey(certBundle.getKey())
+                .withTlsCert(certBundle.getCert())
+                .build(), false);
+
+        WebClient webClient = WebClient.create(VertxFactory.create(), new WebClientOptions()
+                .setSsl(true)
+                .setTrustAll(false)
+                .setPemTrustOptions(new PemTrustOptions()
+                        .addCertPath(certBundle.getCaCert().getAbsolutePath()))
+                .setVerifyHost(false));
+
+        testConsole(webClient);
+    }
+
+    @Test
+    @DisabledIfEnvironmentVariable(named = Environment.useMinikubeEnv, matches = "true")
+    void testOpenshiftCertProvider() throws Exception {
+        createTestEnv(new CertSpecBuilder()
+                .withProvider(CertProvider.openshift.name())
+                .build(),
+                false);
+        boolean testSucceeded = false;
+        try {
+            SystemtestsKubernetesApps.deployOcpEnmasseApp(environment.namespace(), kubernetes);
+            OcpEnmasseAppApiClient client = new OcpEnmasseAppApiClient(kubernetes, SystemtestsKubernetesApps.getOcpEnmasseAppEndpoint(environment.namespace(), kubernetes));
+
+            JsonObject request = new JsonObject();
+            request.put("username", user.getUsername());
+            request.put("password", user.getPassword());
+
+            Endpoint messagingEndpoint = kubernetes.getEndpoint("messaging-"+AddressSpaceUtils.getAddressSpaceInfraUuid(addressSpace), environment.namespace(), "amqps");
+            request.put("messagingHost", messagingEndpoint.getHost());
+            request.put("messagingPort", messagingEndpoint.getPort());
+
+            Endpoint mqttEndpoint = kubernetes.getEndpoint("mqtt-"+AddressSpaceUtils.getAddressSpaceInfraUuid(addressSpace), environment.namespace(), "secure-mqtt");
+            request.put("mqttHost", mqttEndpoint.getHost());
+            request.put("mqttPort", Integer.toString(mqttEndpoint.getPort()));
+
+            Endpoint consoleEndpoint = kubernetes.getEndpoint("console-"+AddressSpaceUtils.getAddressSpaceInfraUuid(addressSpace), environment.namespace(), "https");
+            request.put("consoleHost", consoleEndpoint.getHost());
+            request.put("consolePort", consoleEndpoint.getPort());
+
+            TimeoutBudget timeout = new TimeoutBudget(5, TimeUnit.MINUTES);
+            Exception lastException = null;
+            while (!timeout.timeoutExpired()) {
+                try {
+                    log.info("Making request to openshift app {}", request);
+                    JsonObject response = client.test(request);
+                    if(response.containsKey("error")) {
```

Comment:
I see a lot of formatting issues, which my InteliJ does not like it, could you please reformat code? there for example I mean that you forgot to add space between if and '('